bin/kind-load

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eu
+
+if [ $# -eq 1 ]; then
+    cluster="${1:-}"
+else
+    echo "usage: $(basename $0) cluster-name" >&2
+    exit 64
+fi
+
+bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+. $bindir/_tag.sh
+
+tag="$(head_root_tag)"
+org_repo="gcr.io/linkerd-io"
+
+kind load docker-image $org_repo/proxy:$tag --name $cluster
+kind load docker-image $org_repo/controller:$tag --name $cluster
+kind load docker-image $org_repo/web:$tag --name $cluster
+kind load docker-image $org_repo/cni-plugin:$tag --name $cluster
+kind load docker-image $org_repo/debug:$tag --name $cluster
+kind load docker-image $org_repo/cli-bin:$tag --name $cluster
+kind load docker-image $org_repo/grafana:$tag --name $cluster
```

Comment:
a bit more succinct:
```bash
for img in controller grafana proxy web ; do
  bin/kind load docker-image $org_repo/$img:$tag --name=$cluster
done
```

also, `cli-bin`, `cni-plugin`, and `debug` are not required for a normal installation, and are also large images which slow down this process. how about making them optional (or not including them at all)?

Reply:
Oh good call yea I'll remove the non-required ones.